### PR TITLE
Add tests to controllers

### PR DESCRIPTION
**Pull request by principles31**
# Blog app - controllers specs:
- This part of the project aims to set up tests on the controllers you recently added to your Blog app.
- The project requirements implemented in this **PR** are as follow:
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
> If response status was correct.
> If a correct template was rendered.
> If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```
Comment (Munsa1) [inline spec/requests/posts_spec.rb:1]: - [x] In `posts_spec.rb`, there are 4 tests which are failing. kindly fix this issue
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @principles31, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
- [x] In posts_spec.rb, there are 4 tests which are failing. kindly fix this issue
- [x] The requested changes above have been implemented.

**Comment by principles31** [comment]
@Munsa1 Thank you

**Review by descholar-ceo — APPROVED** [review]
Hi @principles31 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Addressed the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@descholar-ceo Thank you for acknowledging this PR.

**Pull request merged**